concrete/src/File/ImportProcessor/SvgSanitizerProcessor.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Concrete\Core\File\ImportProcessor;
+
+use Concrete\Core\Entity\File\Version;
+use Concrete\Core\File\Image\Svg\Sanitizer;
+use Concrete\Core\File\Image\Svg\SanitizerOptions;
+use Concrete\Core\File\Type\Type;
+
+class SvgSanitizerProcessor implements ProcessorInterface
+{
+    /**
+     * SVG sanitizer.
+     *
+     * @var \Concrete\Core\File\Image\Svg\Sanitizer
+     */
+    protected $sanitizer;
+
+    /**
+     * SVG sanitizer options.
+     *
+     * @var \Concrete\Core\File\Image\Svg\SanitizerOptions|null
+     */
+    private $sanitizerOptions;
+
+    /**
+     * Initialize the instance.
+     *
+     * @param \Concrete\Core\File\Image\Svg\Sanitizer $sanitizer
+     */
+    public function __construct(Sanitizer $sanitizer)
+    {
+        $this->sanitizer = $sanitizer;
+    }
+
+    /**
+     * Set the sanitizer options.
+     *
+     * @param \Concrete\Core\File\Image\Svg\SanitizerOptions $options
+     *
+     * @return $this
+     */
+    public function setSanitizerOptions(SanitizerOptions $options)
+    {
+        $this->sanitizerOptions = $options;
+
+        return $this;
+    }
+
+    /**
+     * Get the sanitizer options.
+     *
+     * @return \Concrete\Core\File\Image\Svg\SanitizerOptions
+     */
+    public function getSanitizerOptions()
+    {
+        if ($this->sanitizerOptions === null) {
+            $this->sanitizerOptions = SanitizerOptions::create();
+        }
+
+        return $this->sanitizerOptions;
+    }
+
+    /**
+     * {@inheritdoc}
```

Comment:
These methods probably shouldn't just inheritdoc, they should describe what they are doing to determine whether it should process, or what they are doing when they process